roles/amq_broker/tasks/install.yml

```diff
@@ -56,7 +56,7 @@
 ## download to controller
 - name: Check local download archive path
   ansible.builtin.stat:
-    path: "{{ lookup('env', 'PWD') }}"
+    path: "{{ activemq_local_archive_repository }}"
```

Comment:
@guidograzioli yet another opportunity to remove stat: and use the jinja filter!